src/DssSpell.sol

```diff
@@ -18,37 +18,35 @@ pragma solidity 0.8.16;
 
 import "dss-exec-lib/DssExec.sol";
 import "dss-exec-lib/DssAction.sol";
-import { GemAbstract } from "dss-interfaces/ERC/GemAbstract.sol";
-import { VatAbstract } from "dss-interfaces/dss/VatAbstract.sol";
+import { MCD, DssInstance } from "dss-test/MCD.sol";
+import { DssLitePsmMigrationPhase2, DssLitePsmMigrationConfigPhase2 } from "./dependencies/dss-lite-psm/phase-2/DssLitePsmMigrationPhase2.sol";
 
-interface DssCronSequencerLike {
-    function addJob(address job) external;
-    function removeJob(address job) external;
+interface RwaMultiSwapOutputConduitLike {
+    function clap(address) external;
+    function slap(address) external;
 }
 
-interface RwaLiquidationLike {
-    function bump(bytes32 ilk, uint256 val) external;
+interface PauseLike {
+    function setDelay(uint256 delay_) external;
 }
 
-interface ProxyLike {
-    function exec(address target, bytes calldata args) external payable returns (bytes memory out);
+interface DssCronSequencerLike {
+    function addJob(address job) external;
+    function removeJob(address job) external;
 }
 
 contract DssSpellAction is DssAction {
     // Provides a descriptive tag for bot consumption
     // This should be modified weekly to provide a summary of the actions
-    // Hash: cast keccak -- "$(wget 'https://raw.githubusercontent.com/makerdao/community/74a3cc88948dcd2f945a523e332801f230a790f7/governance/votes/Executive%20vote%20-%20August%2012%2C%202024.md' -q -O - 2>/dev/null)"
+    // Hash: cast keccak -- "$(wget 'https://raw.githubusercontent.com/makerdao/community/8d95eaf1c9eb6722008172504df88bc27f91ed3c/governance/votes/Executive%20vote%20-%20August%2022%2C%202024.md' -q -O - 2>/dev/null)"
     string public constant override description =
-        "2024-08-12 MakerDAO Executive Spell | Hash: 0xfb6fc47a8f36784d9808f1dde73e5573e748369b8485521873947f9250c2bcc9";
+        "2024-08-12 MakerDAO Executive Spell | Hash: 0xe3794c8152d2a1de72080b1fc7d8429a979015b3f41cbe2c26f755724c70951d";
```

Comment:
Please update target date

Reply:
Fixed in [0f27e22](https://github.com/makerdao/spells-mainnet/pull/422/commits/0f27e22b65209f8e17f73d1cc5c365b6481d8067)